packages/sdk-ethers/src/index.ts

```diff
@@ -1,37 +1,55 @@
-import { Contract, Signer, providers, utils, BigNumberish } from "ethers";
+import { Contract, Signer, utils, BigNumberish, BigNumber } from "ethers";
 import erc6551RegistryAbi from "../abis/ERC6551Registry.json";
 import erc6551AccountAbi from "../abis/IERC6551Account.json";
 
 export { erc6551AccountAbi, erc6551RegistryAbi };
 
 export const erc6551RegistryAddress =
-  "0x1472D0f5c6c151df96352Ec271B8dF1093370A7A" as const;
+  "0x02101dfB77FDE026414827Fdc604ddAF224F0921" as const;
 
 export const erc6551AccountImplementationAddress =
-  "0x36963236d915e4e9b5f70677eBD1ea3e69Cfbbd6" as const;
+  "0x2d25602551487c3f3354dd80d76d54383a243358" as const;
```

Comment:
https://docs.tokenbound.org/contracts/deployments

Reply:
These should already be updated on `main` if you rebase :)